test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj

```diff
@@ -35,10 +35,10 @@
 
   <Target Name="CopyTestSqlFiles" AfterTargets="Build">
     <ItemGroup>
-      <_CSharpTestSqlFiles Include="Integration\test-csharp\Database\**\*.*" />
```

Comment:
This folder didn't even exist - guessing it was left over from some other work and never cleaned up.

Reply:
This is from https://github.com/Azure/azure-functions-sql-extension/pull/317/files#diff-1ef1922c8368ea733b52db01d803d4da431c6a250fe37ca710169142665df11a. The Integration\test-csharp\Database\ folder is in the trigger branch although we should probably move those sql files into test\Database too. 

I think this copy is not needed since we already copy here https://github.com/Azure/azure-functions-sql-extension/blob/a1a045e25ded463f8466dab994051e646ffef5d0/samples/samples-csharp/Microsoft.Azure.WebJobs.Extensions.Sql.Samples.csproj#L19-L21

Reply:
Ah, thanks for digging into that!

We do need both copies - since they're from separate projects and the paths are relative to that project. But we can remove the custom target and instead just use the None items like the samples project does which simplifies it a bit.